sdk/bcs/CHANGELOG.md

```diff
@@ -8,44 +8,59 @@
 
 ## 0.6.0
 
+
+```js
```

Comment:
@damirka heads up, never manually modify this file

Reply:
Oh my. Sorry, it shouldn’t be there. Added by accident.